### PR TITLE
Add test for order exploits in CargoData

### PR DESCRIPTION
**Pull request by corp-0**
### Purpose
Most of the code is a copy paste from the debug command version @ThatDan123 did but I feel this is better because you don't have to manually trigger it but just pr to our repo and receive the shame of allowing players to profit by selling shit.

### CODE REVIEW

File: UnityProject/Assets/Tests/Utils.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	{
+		public static bool TryGetScriptableObjectGUID(Type scriptableObjectType, StringBuilder sb, out string assetGUID)
+		{
+			assetGUID = string.Empty;
+			var typeName = scriptableObjectType.Name;
+			string[] assets = AssetDatabase.FindAssets($"t: {typeName}");
+
+			if (assets.Any() == false)
+			{
+				sb.AppendLine($"{typeName}: could not localte {typeName}");
```
Comment (corp-0) [inline UnityProject/Assets/Tests/Utils.cs]:
```suggestion
				sb.AppendLine($"{typeName}: could not locate {typeName}");
```

File: UnityProject/Assets/Tests/Balance/CargoTest.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using Systems.Cargo;
+using NUnit.Framework;
+using UnityEditor;
+
+namespace Tests.Balance
+{
+	public class CargoTest
+	{
+		[Test]
+		public void StonksTest()
```
Comment (Bod9001) [inline UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: does this handle stuff with inventories and populaters?

Comment (corp-0) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: it does not

Comment (Bod9001) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: you feel like adding it or shall I just merge as is?

Comment (corp-0) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: so check if the stuff inside the crate has inventory and then check the price for everything inside said inventory? We have a problem with recursive inventories @_@ I wished we had like a Utils.GetAllObjectsInInventory to find literally everything inside something in recursive mode.

Comment (Bod9001) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: is up to you

Comment (corp-0) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: I can try later. There is no rush to merge this

Comment (Bod9001) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: ok cool

Comment (corp-0) [reply UnityProject/Assets/Tests/Balance/CargoTest.cs:12]: I surrender, I don't know how to read the content of ItemStoragePopulator.

**Pull request merged**